io.github.lukajankovic.ambience.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The metainfo file should be in the upstream repository ideally.

Reply:
It is under the data directory. Is that ok?

Reply:
Yup! I mean it can be anywhere in your source so long as it gets installed in the correct place, but that looks good. Now you have it upstream you can remove it from here but that's not a blocker.